src/plugins/intel_gpu/src/runtime/ocl/ocl_ext.hpp

```diff
@@ -925,6 +925,8 @@ class UsmMemory {
     // Get methods returns original pointer allocated by openCL.
     void* get() const { return _usm_pointer->ptr(); }
 
+    const UsmHelper& getUsmHelper() { return const_cast<UsmHelper&>(_usmHelper); }
```

Comment:
we may not need to expose this if we could figure out a better solution for [this](https://github.com/slyalin/openvino/pull/169/files#r1747050213)